set.mm

```diff
@@ -666258,6 +666624,671 @@ is real and it is less than or equal to its inferior limit (in such a
+      EWMAERWPWSWQWRUWPCUVOUUPWOWTCUVOUVRXAXKXBXCAGUVIUVIGQZAUVIUWDGUVIUWCQUVIU
+      WDQUVIUVHUEXDXEUVIUWCUWAXFXGMXHZWSXIAGPWGUWLXJXLZWGGWHXMXNZXCXOAUURCFAUUK
+      FQZSZUUNUUQUXBUUNSZUUKXPQZUUQAUXDUUQUXAUUNAUXDSZHUWMUUPTNUXEUWMUVIUUPAUWM
+      XPQUXDAUWMAGPUWMUWLUWTXOXQUOUXEUVHAUVHRQUXDAUVHADIXRXSUOXTUXEUUOUXEUUKDUX
+      DUULAUUKYBYAAUWJUXDIUOZVSXTUXEUWOUWQUWMUVITUGAUWOUXDUWSUOUWRGUVIYLXKUXEDU
+      UKUXFAUXDUMYCYDYEYFUXCUXDVEZUUKEQZUUQUXAUXGUXHAUUNUXAUUKXPEVCZQUXGUXHFUXI
+      UUKLVDUUKXPEVHYGYHAUUNUXHUUQUXAAUUNSUXHSZHUWMUUPTNUXJUWOUUPGQZUWMUUPTUGAU
+      WOUUNUXHUWSVRUUNUXHUXKAUUNUXHSZUUPUWDGUXLUUPUWAQUUPUWDQUXLUUPOUVOUVDDUCUD
+      ZUEUFZUSZVBZUWAUXLUWFUUPUVRQUUPUXPQUXLUUKUVMUVNUXLERUUKUUNUXHUMUULUUMUXHV
+      AYIUUMUUKUVNQVEUULUXHUUKDVGVOYJUUPUUOUEXDXEOUUKUXOUVRUVOUUPUVDUUKULZUXNUU
+      PUXQUXMUUOUEUVDUUKDUCYKYMYNZYOXKOCUVOUXOUVRUXRYPYQUUPUWAUWCYRYSMUUCVNGUUP
+      YLYTYEUUDUUEUUFUUAUUGUVCUUSBHPUUTHULZUVBUURCFUXSUVAUUQUUNUUTHUUPTUUHUUIUU
+      BUUJYT $.
+  $}
+
+  ${
+    $d A w x y $.  $d A w x z $.  $d B w x z $.  $d C w x y $.  $d ph w $.
+    cnrefiisp.a $e |- ( ph -> A e. CC ) $.
+    cnrefiisp.n $e |- ( ph -> -. A e. RR ) $.
+    cnrefiisp.b $e |- ( ph -> B e. Fin ) $.
+    cnrefiisp.c $e |- C = ( RR u. B ) $.
+    $( A non-real, complex number is an isolated point w.r.t. the union of the
+       reals with any finite set (the extended reals is an example of such a
+       union) (Contributed by Glauco Siliprandi, 5-Feb-2022.) $)
+    cnrefiisp $p |- ( ph -> E. x e. RR+ A. y e. C ( ( y e. CC /\ y =/= A ) -> x
+       <_ ( abs ` ( y - A ) ) ) ) $=
+      ( vw vz cv cc cmin co cabs cfv cle csn wcel wne wbr wral crp wrex cim cin
+      wa cdif ciun cun cxr clt cinf eqid weq oveq1 fveq2d sneqd cbviunv infeq1i
+      wi uneq2i cnrefiisplem eleq1w neeq1 anbi12d breq2d imbi12d cbvralv rexbii
+      sylib ) AKMZNUAZVNDUBZUIZBMZVNDOPZQRZSUCZVCZKFUDZBUEUFCMZNUAZWDDUBZUIZVRW
+      DDOPZQRZSUCZVCZCFUDZBUEUFABKDEFDUGRQRTZKENUHDTUJZVTTZUKZULZWMLWNLMZDOPZQR
+      ZTZUKZULZUMUNUOGHIJWQUPUMXCWQUNXBWPWMLKWNXAWOLKUQZWTVTXDWSVSQWRVNDOURUSUT
+      VAVDVBVEWCWLBUEWBWKKCFKCUQZVQWGWAWJXEVOWEVPWFKCNVFVNWDDVGVHXEVTWIVRSXEVSW
+      HQVNWDDOURUSVIVJVKVLVM $.
+  $}
+
+  ${
+    $d A j k u $.  $d F j k u $.  $d M j u $.  $d Z j k u $.  $d j ph $.
+    xlimxrre.m $e |- ( ph -> M e. ZZ ) $.
+    xlimxrre.z $e |- Z = ( ZZ>= ` M ) $.
+    xlimxrre.f $e |- ( ph -> F : Z --> RR* ) $.
+    xlimxrre.a $e |- ( ph -> A e. RR ) $.
+    xlimxrre.c $e |- ( ph -> F ~~>* A ) $.
+    $( If a sequence ranging over the extended reals converges w.r.t. the
+       standard topology on the complex numbers, then there exists an upper set
+       of the integers over which the function is real-valued (the weaker
+       hypothesis ` F e. dom ~~> ` is probably not enough, since in principle
+       we could have +oo e.  CC and -oo e.  CC ) (Contributed by Glauco
+       Siliprandi, 5-Feb-2022.) $)
```

Comment:
Surround +oo e.  CC and -oo e.  CC with grave accents (math mode).

Reply:
Fixed in the latest commit